src/server/transaction.h

```diff
@@ -242,7 +241,8 @@ class Transaction {
   void StartMultiNonAtomic();
 
   // Unlock key locks of a multi transaction.
-  void UnlockMulti();
+  // If block is set, wait for unlock to finish.
```

Comment:
seems that you do not use this argument in prod code. 
you introduced it only for tests?

Reply:
Yes, I will use it in the future, I just split my poc into PRs